sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 ### Bugs Fixed
 
+- On Node, the `timeout` parameter of the `PipelineRequestOptions` object will now only begin counting once the underlying socket is connected. If a socket never connects, a manual timeout will be triggered.
+
```

Comment:
The wording here is confusing to me. What do you mean by a manual timeout? That makes it sound like the customer needs to manually trigger the timeout, but I don't see how they would do that.

It sounds like what you're trying to do is support both a timeout for when the connection is established, and an overall request timeout...is that accurate?

Reply:
Specifically about my "manual" wording, I meant that if we can't setup a timeout specific to the request/socket, we do the setTimeout ourselves 🤔